src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,41 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { ConfigurationService } from '~/app/shared/api/configuration.service';
+import { OrchestratorService } from '~/app/shared/api/orchestrator.service';
+import { SummaryService } from '~/app/shared/services/summary.service';
 
 @Component({
   selector: 'cd-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
-export class DashboardComponent {}
+export class DashboardComponent implements OnInit {
+  detailsCardData: any = {};
```

Comment:
i think its better to have an interface here?

```
export interface DashboardDetails {
    fsid: string
    ...
}
```

Reply:
Hmmm...It would make sense, I'll change it

Reply:
Added  the interface